server.js

```diff
@@ -17,115 +26,119 @@ const options = {
 }
 // ☞ bd397750-457f-4308-b616-f0424ddc5d04
 
-mongoose.connect('mongodb://ds141611.mlab.com:41611/d2rd-notes', options)
-.then(() => console.log('Success connecting the MongoDB/d2rd-notes on mlab'))
+mongoose.connect(activeDB, options)
+.then(() => console.log('Success connecting the MongoDB/notes on mlab'))
 .catch((err) => console.log(err.message)) // TEST: changing PW should throw 'authentication failed error
-
+// NOTE: EACH DB HAS A UNIQUE CONNECTION STRING
 // create schema
 // ☞ 8cf866c9-a061-48df-a275-ebdbf2196f60
 // REFACTORED TO MOVE NOTES TO MONGODB
 
+//TEST DATA
+const newTestNote = {
```

Comment:
you can use faker.js to bring in some fake data so you don't have to do this...and it would do it at scale too.

Reply:
Cool thanks!